modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/prepare/pruning/PartitionPruningPredicateSelfTest.java

```diff
@@ -118,12 +120,20 @@ public void testLiteralValue(ColumnType columnType) {
         expectPartitionsPruned(table, columns, new Object[0], group, val);
     }
 
+    private static NativeType getNativeType(ColumnType columnType) {
+        SqlTypeName sqlTypeName = SqlTestUtils.columnType2SqlTypeName(columnType);
+        int precision = IgniteTypeSystem.INSTANCE.getMaxPrecision(sqlTypeName);
+        int scale = IgniteTypeSystem.INSTANCE.getMaxScale(sqlTypeName);
+
+        return TypeUtils.columnType2NativeType(columnType, precision, scale, 4);
```

Comment:
The length parameter should have the same value as the precision (because they are the same thing).

Reply:
agree. Fixed